File: Modul-3 (Ibnu)/SOAL 4/PRAK304-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main(){
    int Wilangan;

    printf("Input:\n ");
    scanf("%d", &Wilangan);

    printf("Output:\n");
    if(Wilangan>=100){
    printf("Anda Menginput Bilangan Melebihi Limit");
    }else if(Wilangan>1 && Wilangan<10){
        printf("Satuan");
    }else if (Wilangan>9 && Wilangan<20){
        printf("Belasan");
    }else if(Wilangan>=20 && Wilangan<100){
        printf("Puluhan");
    }else {
        printf("Nol");
    }
}

File: Modul-3 (Ibnu)/SOAL 4/PRAK304-2310817210009-MuhammadIbnuSina.py
Wilangan = int(input("Input : \n"))

print ("output : ")
if(Wilangan >= 100) :
    print("Anda Menginput Melebihi Limit Bilangan")
elif(Wilangan > 1 and Wilangan < 10) :
    print("Satuan")
elif (Wilangan > 9 and Wilangan < 20) :
    print("Belasan")
elif(Wilangan >= 20 and Wilangan < 100) :
    print("Puluhan")
else :
    print("Nol")